daemon/api_quotas.go

```diff
@@ -52,9 +54,11 @@ type postQuotaGroupData struct {
 	// Action can be "ensure" or "remove"
 	Action    string   `json:"action"`
 	GroupName string   `json:"group-name"`
-	MaxMemory uint64   `json:"max-memory,omitempty"`
 	Parent    string   `json:"parent,omitempty"`
 	Snaps     []string `json:"snaps,omitempty"`
+	// Constraints is a map of resource type to constraint, currently accepted
+	// resource types is just "memory"
+	Constraints map[string]interface{} `json:"constraints,omitempty"`
```

Comment:
can't this use *client.QuotaValues  ?

Reply:
sorry missed this one when cooking up the changes, I will fix this shortly and push it up

Reply:
this is updated now